src/OrbitQt/main.cpp

```diff
@@ -342,6 +373,23 @@ int main(int argc, char* argv[]) {
     return -1;
   }
 
-  RunUiInstance(deployment_configuration, &context.value());
+  std::vector<std::string> command_line_args;
+  if (argc > 1) {
+    command_line_args.assign(argv + 1, argv + argc);
+  }
```

Comment:
I fear this is too late for making a copy. `argc` and `argv` are passed into the QApplication constructor in line 308 and might be changed then:

From https://doc.qt.io/qt-5/qapplication.html#QApplication
> Note: argc and argv might be changed as Qt removes command line arguments that it recognizes.

Making a copy earlier should solve that

Reply:
Nice catch! Thanks so much for pointing out this!